suggestions/sandboxing_apps.json

```diff
@@ -3,6 +3,14 @@
         "id": "com.oasisfeng.island.fdroid",
         "label": "Insular",
         "source": "f",
+        "reason": "Isolate your Big Brother apps",
         "repo": "https://gitlab.com/secure-system/Insular"
+    },
+    {
+        "id": "net.typeblog.shelter",
+        "label": "Shelter",
+        "source": "f",
+        "reason": "Isolate and run multiple instances of apps using Work Profiles",
+        "repo": "https://gitea.angry.im/PeterCxy/Shelter"
```

Comment:
Requires a review: https://secure-system.gitlab.io/Insular/faq.html

@shuvashish76: Any opinions?

Reply:
Seems to have fewer features that Insular. (Note that Insular is partially inspired by Shelter, so it should have nothing to envy)

Reply:
Insular also has freeze feature.
The most important feature, I noticed so far is "File Shuttle" feature of Shelter which is missing in Insular.
![Screenshot_2023-10-06_22-17-04_1](https://github.com/MuntashirAkon/android-debloat-list/assets/48226312/11aabacc-1932-453a-b7a1-3d9d17edabf1)

Reply:
> Insular also has freeze feature.

Official guide (linked above) says otherwise.

Reply:
OK no Auto-freeze feature 👍🏿.